games-simulation/freerct/freerct-0.2~git.recipe

```diff
@@ -31,31 +36,55 @@ REQUIRES="
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libGLEW$secondaryArchSuffix
+	devel:libglfw$secondaryArchSuffix
 	devel:libpng16$secondaryArchSuffix
 	devel:libSDL2_2.0$secondaryArchSuffix
 	devel:libSDL2_ttf_2.0$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
+	cmd:bison
 	cmd:cmake
+	cmd:doxygen
```

Comment:
doxygen without cmd:dot probably won't produce any documentation, so either drop doxygen or add dot to see if any documentations is actually build?

Reply:
I'll have a look, from experience I've seen this failing when dot wasn't added.

Reply:
I was right on the first one ;)
```
-- Found Doxygen: /bin/doxygen (found version "1.12.0") found components: doxygen missing components: dot
```

Reply:
OK, you can build the documentation with adding `cmd:dot` here and use `make -C build doc`, manual installations for it is needed then (doc/html to $docDir).
However looking at the generated documentation it's mostly concerned around the source usage for the package, I doublt any user will be looking for it, so my 2 cents, drop `cmd:doxygen` and just forget about the documentation in the package.